playwright/tests/screenshots/editor/editor.spec.ts

```diff
@@ -40,6 +41,28 @@ test('extensions', async ({ page }) => {
   await screenshot(page, 'extensions.png');
 });
 
+test('connector-process', async ({ page }) => {
+  const editor = await ProcessEditor.openProcess(page, { file: '/processes/market/erp.p.json', waitFor: '.sprotty-graph' });
+  await editor.element('start:callSubStart').inscribe();
+  const inscription = new Inscription(page);
+  inscription.waitForView();
```

Comment:
editor.element().inscribe returns the inscription page object or not?!

Reply:
yep, you are right ... made it simpler